tests/src/dataValues.DataValue.tests.js

```diff
@@ -124,7 +124,7 @@ define( [
 					'is instance of actual data value implementation\'s constructor'
 				);
 				assert.equal(
-					typeof( instance.getType() ),
+					typeof ( instance.getType() ),
```

Comment:
`typeof` is not a function. The round brackets should be removed.

Same for all `typeof` below.